test-plugins/activity-container/local-plugin.kdl

```diff
@@ -0,0 +1,15 @@
+publisher "mitre"
+name "activity-container"
+version "0.0.0"
+license "Apache-2.0"
+
+entrypoint {
+  on arch="aarch64-apple-darwin" "activity-container-deploy.sh"
+  on arch="x86_64-apple-darwin" "activity-container-deploy.sh"
+  on arch="x86_64-unknown-linux-gnu" "activity-container-deploy.sh"
+  on arch="x86_64-pc-windows-msvc" "activity-container-deploy.sh"
```

Comment:
Is Windows able to run this shell script? You may need to also have a PowerShell script for Windows specifically.